QUEST.tsv

```diff
@@ -92,10 +92,10 @@ xQUEST_20150317_000091	And yet, it doesn't matter. Come out and face me, you mon
 xQUEST_20150317_000092	Just now, I saw a goddess appear in my dream… {nl}How surreal…
 xQUEST_20150317_000093	Still. I have a duty to defend the kingdom. It's why I'm here now.
 xQUEST_20150317_000094	Uska's an old friend of mine.
-xQUEST_20150317_000095	We did our tests together, and Uska got to where he is now with my support. {nl}Oh, those were some crazy days… It'd be nice to get together again!
-xQUEST_20150317_000096	Just don't believe some of the things he'll say. {nl}His hometown knew him for being quite the show-off.
+xQUEST_20150317_000095	We did our tests together, and Uska only got to where he is now with my help. {nl}Oh, those were some crazy days… It'd be nice to get together again!
```

Comment:
Remove the comma
Remove only (It is not implied that its "only" because of he/she's help.)
Change help back to support